### ISSUE TITLE
ite: eSPI driver: espi_it8xxx2_send_vwire() is not setting valid flag along with respective virtual wire when invoked from app code.

### ISSUE THREAD

**Issue by NagarajuGundimi** [zephyrproject-rtos/zephyr#52776]
<img width="886" alt="espi_it8xxx2_send_vwire_issue" src="https://user-images.githubusercontent.com/85330759/205558774-8d378102-a7d5-4588-9c91-be4dfd15128b.png">

**Describe the bug**
ite: eSPI driver: espi_it8xxx2_send_vwire() is not setting valid flag along with respective virtual wire when invoked from app code.

When any virtual wire is set/clear from app code using espihub_send_vw() API it is invoking espi_it8xxx2_send_vwire(). But espi_it8xxx2_send_vwire() is only setting respective virtual wire and NOT corresponding valid flag. Even though espi_vw_set_valid() function is available there is no API to invoke it from app code.

**Expected behavior**
espi_it8xxx2_send_vwire() should also set respective valid flag along with virtual wire.

**Impact**
Not able to boot to OS(Windows) in MAFS mode.

**Comment by albertofloyd** [comment]
@Dino-Li @GTLin08 could you take a look into this?

**Comment by Dino-Li** [comment]
Loop BJ @itebj

**Comment by itebj** [comment]
Hi sirs, 

I was just looking into this and I think that it's about the current MAFS under experiments with the Intel and I think that this item has been already resolved after the discussions yesterday with @NagarajuGundimi. We've also applied the changes to the current MAFS code.

@NagarajuGundimi 
According to your suggestions, I've modified and tested the new App code in espi flash and now it can directly boot after the MAFS mirroring. Now it works well after the direct jumping to the App code. It can boot all the way to the OS.

Thanks again for great patience.
